### PR TITLE
[daap] remove daap support

### PR DESCRIPTION
**Pull request by stefansaraev**
as discussed in forum

//cc @mkortstiege @Memphiz

### CODE REVIEW

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -10153,7 +10153,7 @@ msgctxt "#20173"
 msgid "FTP server"
 msgstr ""
 
-#: xbmc/network/GUIDialogNetworkSetup.cpp
+#: unused
```
Comment (mkortstiege) [inline addons/resource.language.en_gb/resources/strings.po]: Please remove the entire string

**Comment by mkortstiege** [comment]
Looks good once the string is completely nuked. Thanks.

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (stefansaraev) [inline addons/resource.language.en_gb/resources/strings.po]: I read somewhere that strings should never be removed. I'll do.

Comment (MartijnKaijser) [inline addons/resource.language.en_gb/resources/strings.po]: because some skins could be using them them though it's unlikely this one is used

**Comment by MartijnKaijser** [comment]
jenkins build this please

File: xbmc/utils/test/TestURIUtils.cpp [hunk 2]
```diff
@@ -358,11 +353,6 @@ TEST_F(TestURIUtils, IsOnDVD)
 
 TEST_F(TestURIUtils, IsOnLAN)
 {
-  std::vector<std::string> multiVec;
-  multiVec.push_back("daap://path/to/file");
-  EXPECT_TRUE(URIUtils::IsOnLAN(CMultiPathDirectory::ConstructMultiPath(multiVec)));
-  EXPECT_TRUE(URIUtils::IsOnLAN("stack://daap://path/to/file"));
```
Comment (notspiff) [inline xbmc/utils/test/TestURIUtils.cpp]: these tests arent testing anything related to daap as such. make it smb:// instead.

File: xbmc/filesystem/CurlFile.cpp [hunk 3]
```diff
@@ -1214,9 +1214,6 @@ int64_t CCurlFile::Seek(int64_t iFilePosition, int iWhence)
   // re-setup common curl options
   SetCommonOptions(m_state);
 
-  /* caller might have changed some headers (needed for daap)*/
-  SetRequestHeaders(m_state);
-
```
Comment (notspiff) [inline xbmc/filesystem/CurlFile.cpp]: are you certain it's only needed for daap?

**Comment by stefansaraev** [comment]
thanks for the comments @notspiff. it seems I overlooked it a bit. comments addressed.

I'll squash when this is ready for merging

**Comment by MartijnKaijser** [comment]
jenkins build and merge

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Pull request merged**